{
    "repository": "speechbrain/speechbrain",
    "stars": 6205,
    "language": "python",
    "size": 74813,
    "clone_url": "https://github.com/speechbrain/speechbrain.git",
    "timestamp": "2023-07-01T14:50:35.114529Z",
    "clone_success": true,
    "number_of_actions": 5,
    "number_of_test_actions": 1,
    "actions_successful": false,
    "actions_build_tools": [
        "unknown",
        "unknown",
        "unknown",
        "unknown",
        "pytest"
    ],
    "actions_test_build_tools": [
        "pytest"
    ],
    "actions_run": {
        "failed": true,
        "tests": [],
        "stdout": "[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests] \ud83d\ude80  Start image=crawlergpt:latest\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udc33  docker pull image=crawlergpt:latest platform= username= forcePull=false\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udc33  docker create image=crawlergpt:latest platform= entrypoint=[\"tail\" \"-f\" \"/dev/null\"] cmd=[]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udc33  docker run image=crawlergpt:latest platform= entrypoint=[\"tail\" \"-f\" \"/dev/null\"] cmd=[]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udc33  docker exec cmd=[chown -R 1012:1013 /tmp/558156d8-17ed-11ee-8a50-bb14de238602/speechbrain-speechbrain] user=0 workdir=\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \u2601  git clone 'https://github.com/actions/setup-python' # ref=v1\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests] \ud83e\uddea  Matrix: map[python-version:3.7]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests] \u2b50 Run Main actions/checkout@v2\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \u2705  Success - Main actions/checkout@v2\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests] \u2b50 Run Main Set up Python 3.7\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udc33  docker cp src=/tmp/act-cache/81906e92-8a25-4e1f-b1a5-7b2add21cd32/act/actions-setup-python@v1/ dst=/var/run/act/actions/actions-setup-python@v1/\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udc33  docker exec cmd=[chown -R 1012:1013 /var/run/act/actions/actions-setup-python@v1/] user=0 workdir=\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udc33  docker exec cmd=[node /var/run/act/actions/actions-setup-python@v1/dist/index.js] user= workdir=\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udcac  ::debug::Semantic version spec of 3.7 is 3.7\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udcac  ::debug::isExplicit: \n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udcac  ::debug::explicit? false\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udcac  ::debug::isExplicit: 2.7.18\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udcac  ::debug::explicit? true\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udcac  ::debug::isExplicit: 3.5.10\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udcac  ::debug::explicit? true\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udcac  ::debug::isExplicit: 3.6.14\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udcac  ::debug::explicit? true\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udcac  ::debug::isExplicit: 3.7.11\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udcac  ::debug::explicit? true\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udcac  ::debug::isExplicit: 3.8.11\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udcac  ::debug::explicit? true\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udcac  ::debug::isExplicit: 3.9.6\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udcac  ::debug::explicit? true\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udcac  ::debug::evaluating 6 versions\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udcac  ::debug::matched: 3.7.11\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udcac  ::debug::checking cache: /opt/hostedtoolcache/Python/3.7.11/x64\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udcac  ::debug::Found tool in cache Python 3.7.11 x64\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Successfully setup CPython (3.7.11)\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \u2753 add-matcher /run/act/actions/actions-setup-python@v1/.github/python.json\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \u2705  Success - Main Set up Python 3.7\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \u2699  ::set-env:: pythonLocation=/opt/hostedtoolcache/Python/3.7.11/x64\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \u2699  ::set-output:: python-version=3.7.11\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \u2699  ::add-path:: /opt/hostedtoolcache/Python/3.7.11/x64\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \u2699  ::add-path:: /opt/hostedtoolcache/Python/3.7.11/x64/bin\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests] \u2b50 Run Main Install libsndfile\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \ud83d\udc33  docker exec cmd=[bash --noprofile --norc -e -o pipefail /var/run/act/workflow/2] user= workdir=\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Ign:1 https://dl.hhvm.com/ubuntu focal InRelease\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Err:2 https://dl.hhvm.com/ubuntu focal Release\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   |   Certificate verification failed: The certificate is NOT trusted. The certificate issuer is unknown.  Could not handshake: Error in the certificate verification. [IP: 108.157.214.102 443]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Get:3 https://cloud.r-project.org/bin/linux/ubuntu focal-cran40/ InRelease [3622 B]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Hit:4 http://ppa.launchpad.net/ubuntu-toolchain-r/test/ubuntu focal InRelease\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Get:5 https://packages.microsoft.com/ubuntu/20.04/prod focal InRelease [3611 B]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Get:6 http://dl.google.com/linux/chrome/deb stable InRelease [1825 B]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Hit:7 http://archive.ubuntu.com/ubuntu focal InRelease\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Get:8 http://security.ubuntu.com/ubuntu focal-security InRelease [114 kB]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Get:9 http://archive.ubuntu.com/ubuntu focal-updates InRelease [114 kB]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Get:10 https://cloud.r-project.org/bin/linux/ubuntu focal-cran40/ Packages [81.0 kB]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Hit:11 https://adoptopenjdk.jfrog.io/adoptopenjdk/deb focal InRelease\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Get:12 http://archive.ubuntu.com/ubuntu focal-backports InRelease [108 kB]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Get:13 https://packages.microsoft.com/ubuntu/20.04/prod focal/main amd64 Packages [211 kB]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Err:6 http://dl.google.com/linux/chrome/deb stable InRelease\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   |   The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 4EB27DB2A3B88B8B\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Get:14 https://packages.microsoft.com/ubuntu/20.04/prod focal/main all Packages [2521 B]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Get:15 http://security.ubuntu.com/ubuntu focal-security/universe amd64 Packages [1066 kB]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Get:16 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages [3336 kB]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Get:17 http://security.ubuntu.com/ubuntu focal-security/main i386 Packages [764 kB]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Get:18 http://security.ubuntu.com/ubuntu focal-security/multiverse amd64 Packages [28.5 kB]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Get:19 http://security.ubuntu.com/ubuntu focal-security/restricted i386 Packages [36.9 kB]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Get:20 http://security.ubuntu.com/ubuntu focal-security/universe i386 Packages [744 kB]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Get:21 http://security.ubuntu.com/ubuntu focal-security/multiverse i386 Packages [8141 B]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Get:22 http://security.ubuntu.com/ubuntu focal-security/main amd64 Packages [2856 kB]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Get:23 http://security.ubuntu.com/ubuntu focal-security/restricted amd64 Packages [2479 kB]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Get:24 http://archive.ubuntu.com/ubuntu focal-updates/multiverse amd64 Packages [39.5 kB]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Get:25 http://archive.ubuntu.com/ubuntu focal-updates/universe amd64 Packages [1366 kB]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Get:26 http://archive.ubuntu.com/ubuntu focal-updates/restricted amd64 Packages [2636 kB]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Get:27 http://archive.ubuntu.com/ubuntu focal-backports/main amd64 Packages [55.2 kB]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Get:28 http://archive.ubuntu.com/ubuntu focal-backports/universe amd64 Packages [28.6 kB]\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | Reading package lists...\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | E: The repository 'https://dl.hhvm.com/ubuntu focal Release' no longer has a Release file.\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   | W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: http://dl.google.com/linux/chrome/deb stable InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 4EB27DB2A3B88B8B\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests]   \u274c  Failure - Main Install libsndfile\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests] exitcode '100': failure\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests] failed to remove container: Delete \"http://%2Fvar%2Frun%2Fdocker.sock/v1.41/containers/e6cb8e7dc0d57ce4e8580c1146692cfe28ba703c6f3028b0bff87233964619d1?force=1&v=1\": context deadline exceeded\n[50b3a0d0-7ec3-4294-bf6c-c0ea985757f2/Tests] \ud83c\udfc1  Job failed\n",
        "stderr": "Error: Error occurred running finally: Error occurred running finally: context deadline exceeded (original error: <nil>) (original error: <nil>)\n",
        "workflow": {
            "path": "/tmp/558156d8-17ed-11ee-8a50-bb14de238602/speechbrain-speechbrain/.github/workflows/pythonapp-crawler.yml",
            "type": "pytest"
        },
        "workflow_name": "50b3a0d0-7ec3-4294-bf6c-c0ea985757f2",
        "build_tool": "pytest",
        "elapsed_time": 277.0010712146759
    }
}